packages/manager/apps/pci-kubernetes/src/api/hooks/useAvailableRegions.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { renderHook, waitFor } from '@testing-library/react';
+import { describe, it, vi } from 'vitest';
+import * as ApiAvailableRegionsModule from '@/api/data/available-regions';
+import { TAvailableRegionsResponse } from '@/api/data/available-regions';
+import { useAvailableRegions } from '@/api/hooks/useAvailableRegions';
+import { wrapper } from '@/wrapperRenders';
+
+vi.mock('@/api/data/available-regions', async () => {
+  const mod = await vi.importActual('@/api/data/available-regions');
+  return {
+    ...mod,
+    getAvailableRegions: vi.fn(),
+  };
+});
```

Comment:
You dont need the mock of the module here, since you spyOn the needed funtions bellow